Userland/Applications/Help/MainWidget.cpp

```diff
@@ -183,47 +183,22 @@ MainWidget::MainWidget()
     };
 }
 
-void MainWidget::set_start_page(StringView start_page, u32 section)
+void MainWidget::set_start_page(StringView first_query_parameter, StringView optional_second_query_parameter)
```

Comment:
Hmm, this sounds both awfully hardcoded and vague at the same time. Could we instead accept a `Vector<StringView>` here which would then be passed directly to `LibManual`? This would additionally remove the need to change this part for future argument parsing changes in `LibManual`.